From: James Rowe <jnrowe@gmail.com>
Subject: Workaround relative path breakage
Status: LOCAL

This fixes the path mess caused by a combination of the automagic path guessing
in pytrainer and the Three-card Monte that is python-exec.  This is most
definitely not intended for upstreaming.

diff --git i/bin/pytrainer w/bin/pytrainer
index a28cf628b879..52087b75d58d 100755
--- i/bin/pytrainer
+++ w/bin/pytrainer
@@ -63,23 +63,13 @@ def _max(one, two):
 
 
 bin_path = os.path.realpath(os.path.dirname(__file__)) # directory that the pytrainer script executes from e.g. /usr/bin or /usr/local/bin
-base_path = os.path.dirname(bin_path)
+base_path = sys.prefix
 #Get the version of the running python interpreter
 ver = platform.python_version_tuple()
 
-if (os.path.exists(base_path + "/INSTALL") 
-    and os.path.exists(base_path + "/setup.py") 
-    and os.path.exists(base_path + "/pytrainer/main.py")
-    and os.path.exists(base_path + "/locale")):
-    print("running pytrainer from source path")
-    data_path = base_path + "/"
-    site_path = base_path
-    gettext_path = base_path + "/locale"
-else:
-    print("running pytrainer from egg installation")
-    data_path = base_path + "/share/pytrainer/"
-    site_path =  "%s/lib/python%s.%s/site-packages" % (base_path, ver[0], ver[1])
-    gettext_path = base_path + "/share/locale"
+data_path = base_path + "/share/pytrainer/"
+site_path =  "%s/lib/python%s.%s/site-packages" % (base_path, ver[0], ver[1])
+gettext_path = base_path + "/share/locale"
 
 print "data_path: " + data_path
 print "gettext_path: " + gettext_path
